che-theia-java-extension/package.json

```diff
@@ -9,24 +9,29 @@
     "src"
   ],
   "dependencies": {
-    "@theia/core": "0.3.13",
-    "@theia/java": "0.3.13",
-    "@theia/editor": "0.3.13",
-    "@theia/languages": "0.3.13"
+    "@theia/core": "0.3.15",
+    "@theia/java": "0.3.15",
+    "@theia/editor": "0.3.15",
+    "@theia/languages": "0.3.15",
+    "@theia/filesystem": "0.3.15",
+    "@theia/navigator": "0.3.15",
+    "@theia/workspace": "0.3.15"
   },
   "devDependencies": {
     "rimraf": "2.6.2",
-    "typescript": "^2.7.2"
+    "typescript": "2.9.2"
```

Comment:
You removed the '^'. Why?

Reply:
Restored

Reply:
note that you should not use `^` as Eclipse CQs are only validated for a given version (and not a range)

Reply:
Thanks @benoitf will change it back to `2.9.2` :)